utils/tinystr/src/ascii.rs

```diff
@@ -720,6 +856,78 @@ mod test {
         check::<16>();
     }
 
+    #[test]
+    fn test_is_ascii_alphabetic_lowercase() {
+        fn check<const N: usize>() {
+            check_operation(
+                |s| {
+                    // Convert alphabetic
```

Comment:
Nit (here and below): "Check alphabetic" and "Check lowercase" rather than "Convert ..."